en/docs/visual-programming/sequence-diagram-view.md

```diff
@@ -1,24 +1,28 @@
-# Sequence diagram view
-With Ballerina, you can view your code in a graphical mode and develop your program logic through it. This
-allows you to look at your program logic holistically and gives you the ability to comprehend the program at 
-a glance. This visual representation has its roots in sequence diagrams and can capture how the logic of
-the program flows, how the concurrent execution flow works, which remote endpoints are involved, and how they interact 
-with the different objects in the program.
+# Diagram view
+
+When you use the Ballerina VS Code extension, you can view your code in graphical mode and develop your program logic through it. This
+allows you to look at your program logic holistically and comprehend the program at a glance. 
+
+This visual representation has its roots in sequence diagrams and can capture how the logic of your program flows, how the concurrent execution flow works, which remote endpoints are involved, and how those endpoints interact with the different objects in the program.
 
 ## Switch to graphical mode
+
 There are two methods for switching to the graphical mode using the VS Code extension.
 
-1. Execute the `Ballerina: Show Diagram` command from the command palette.
+-   Execute the `Ballerina: Show Diagram` command from the command palette.
+    
     ![command screenshot](../img/show-diagram-command-palette.png)
 
-2. Click the `Show Diagram` button in the editor group.
+-   Click `Show Diagram` in the editor group.
+    
     ![button screenshot](../img/editor-group-button.png)
 
 ## Graphical representation of code
 
-Ballerina code can be represented graphically in the following ways.
+See the topics given below to learn how various elements of the Ballerina code are represented graphically.
```

Comment:
`See the topics below` seems a bit unnecessary to me as it is always sub-topics and a common thing. No need to specifically say `topics`